tox.ini

```diff
@@ -31,6 +31,19 @@ filterwarnings =
     #
     ignore:^Use of \.\. or absolute path in a resource path is not allowed and will raise exceptions in a future release\.$:DeprecationWarning:pkg_resources
 
+    # Ignore WebOb warnings that just say "<method> will be changing in the
+    # future" and don't say how it will be changing or what developers can do
+    # now to avoid the warning. I don't think these warnings _can_ be avoided
+    # currently.
+    ignore:^The behavior of \.best_match for the Accept classes is currently being maintained for backward compatibility, but the method will be deprecated in the future, as its behavior is not specified in \(and currently does not conform to\) RFC 7231\.$:DeprecationWarning:webob.acceptparse
```

Comment:
It looks like Accept handling was overhauled in the [latest Pyramid releases](https://github.com/Pylons/pyramid/pull/3326). Does Pyramid 1.10 fix any of these warnings?

Reply:
Oh - these are warnings triggered directly by our code as opposed to by Pyramid?